devito/compiler.py

```diff
@@ -32,7 +45,20 @@ def __str__(self):
 
 
 class GNUCompiler(Compiler):
-    """Set of standard compiler flags for the GCC toolchain"""
+    """Set of standard compiler flags for the GCC toolchain
+
+    :param openmp: boolean indicating if openmp is enabled. False by default
```

Comment:
There is an extension to display the inheritance diagram, but I don't think that's what we want. A possible solution could be to simply specify that the parameters are the same as in the base class since the base class name is displayed automatically on the generated documentation